CMakeLists.txt

```diff
@@ -226,4 +209,296 @@ function(importLibraries)
+      DESTINATION "/control/rpm/etc/sysconfig"
+      RENAME "osqueryd"
+    )
+
+    install(
+      FILES "LICENSE"
+      DESTINATION "/control"
+      RENAME "LICENSE.txt"
+    )
+
+  elseif(PLATFORM_WINDOWS)
+    install(
+      TARGETS osqueryd
+      DESTINATION "osqueryd"
+    )
+
+    install(
+      PROGRAMS "$<TARGET_FILE:osqueryd>"
+      DESTINATION "."
+      RENAME "osqueryi.exe"
+    )
+
+    install(
+      DIRECTORY "tools/deployment/windows_packaging/chocolatey/tools"
+      DESTINATION "/control/nupkg"
+    )
+
+    install(
+      FILES "LICENSE"
+      DESTINATION "/control/nupkg/extras"
+      RENAME "LICENSE.txt"
+    )
+
+    install(
+      FILES "LICENSE"
+      DESTINATION "/control"
+      RENAME "LICENSE.txt"
+    )
+
+    install(
+      FILES "tools/deployment/windows_packaging/chocolatey/VERIFICATION.txt"
+      DESTINATION "/control/nupkg/extras"
+    )
+    
+    install(
+      FILES "tools/deployment/osquery.example.conf"
+      DESTINATION "."
+      RENAME "osquery.conf"
+    )
+
+    install(
+      FILES "tools/wel/osquery.man"
+      DESTINATION "."
+    )
+
+    install(
+      FILES "tools/deployment/windows_packaging/manage-osqueryd.ps1"
+      DESTINATION "."
+    )
+
+    install(
+      FILES "tools/deployment/windows_packaging/osquery_utils.ps1"
+      DESTINATION "."
+    )
+
+    install(
+      FILES "tools/deployment/windows_packaging/osquery.flags"
+      DESTINATION "."
+    )
+
+    install(
+      FILES "tools/deployment/windows_packaging/manage-osqueryd.ps1"
+      DESTINATION "/control/nupkg/extras"
+    )
+
+    install(
+      FILES "tools/deployment/windows_packaging/osquery_utils.ps1"
+      DESTINATION "/control/nupkg/tools"
+    )
+
+    install(
+      FILES "tools/deployment/windows_packaging/osquery.ico"
+      DESTINATION "/control/msi"
+    )
+
+    install(
+      FILES "tools/deployment/windows_packaging/msi/osquery_wix_patch.xml"
+      DESTINATION "/control/msi"
+    )
+
+    install(
+      FILES "tools/deployment/certs.pem"
+      DESTINATION "certs"
+    )
+
+    install(
+      DIRECTORY
+      DESTINATION "log"
+    )
+
+  elseif(PLATFORM_MACOS)
+    install(
+      FILES
+        "tools/deployment/macos_packaging/pkg/productbuild.sh"
+
+      DESTINATION
+        "/control/pkg"
+
+      PERMISSIONS
+        OWNER_READ OWNER_WRITE OWNER_EXECUTE
+        GROUP_READ             GROUP_EXECUTE
+        WORLD_READ             WORLD_EXECUTE
+    )
+
+    install(
+      TARGETS osqueryd
+      DESTINATION "bin"
+    )
+
+    execute_process(
+      COMMAND "${CMAKE_COMMAND}" -E create_symlink osqueryd osqueryi
+      WORKING_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}"
+    )
+
+    install(
+      FILES "${CMAKE_CURRENT_BINARY_DIR}/osqueryi"
+      DESTINATION "bin"
+    )
+
+    install(
+      FILES "tools/deployment/osqueryctl"
+      DESTINATION "bin"
+    )
+
+    install(
+      DIRECTORY "${augeas_lenses_path}"
+      DESTINATION "/private/var/osquery/lenses"
```

Comment:
Can we remove the extraneous lenses here? It will be a regression of #6995 / #6998